[2023-04-26T09:03:32.891+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.copy_task manual__2023-04-26T09:03:25.419117+00:00 [queued]>
[2023-04-26T09:03:32.896+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.copy_task manual__2023-04-26T09:03:25.419117+00:00 [queued]>
[2023-04-26T09:03:32.897+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T09:03:32.897+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T09:03:32.898+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T09:03:32.904+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): copy_task> on 2023-04-26 09:03:25.419117+00:00
[2023-04-26T09:03:32.909+0000] {standard_task_runner.py:55} INFO - Started process 1754 to run task
[2023-04-26T09:03:32.913+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'copy_task', 'manual__2023-04-26T09:03:25.419117+00:00', '--job-id', '227', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp1f_u1ifx']
[2023-04-26T09:03:32.915+0000] {standard_task_runner.py:83} INFO - Job 227: Subtask copy_task
[2023-04-26T09:03:32.962+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.copy_task manual__2023-04-26T09:03:25.419117+00:00 [running]> on host c452347a64f2
[2023-04-26T09:03:33.005+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=copy_task
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T09:03:25.419117+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T09:03:25.419117+00:00
[2023-04-26T09:03:33.007+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-04-26T09:03:33.008+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "PGPASSWORD=*** psql -U *** -h postgres -p 5432 -d *** -c 'COPY users(first_name, last_name,country, username, password, email) FROM /opt/***/csv_files/user_processed.csv delimiter , CSV'"]
[2023-04-26T09:03:33.019+0000] {subprocess.py:86} INFO - Output:
[2023-04-26T09:03:33.048+0000] {subprocess.py:93} INFO - ERROR:  syntax error at or near "/"
[2023-04-26T09:03:33.049+0000] {subprocess.py:93} INFO - LINE 1: ...ast_name,country, username, password, email) FROM /opt/airfl...
[2023-04-26T09:03:33.049+0000] {subprocess.py:93} INFO -                                                              ^
[2023-04-26T09:03:33.050+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2023-04-26T09:03:33.059+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2023-04-26T09:03:33.062+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=user_processing, task_id=copy_task, execution_date=20230426T090325, start_date=20230426T090332, end_date=20230426T090333
[2023-04-26T09:03:33.070+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 227 for task copy_task (Bash command failed. The command returned a non-zero exit code 1.; 1754)
[2023-04-26T09:03:33.086+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-04-26T09:03:33.100+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
